rules/macos/execution_shell_execution_via_apple_scripting.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the Shell process via Apple Scripting (JXA or AppleScript). Adversaries may use
+doShellScript functionality in JXA or do shell script in AppleScript to execute System commands.
```

Comment:
```suggestion
Identifies the execution of the shell process (sh) via scripting (JXA or AppleScript). Adversaries may use
the doShellScript functionality in JXA or do shell script in AppleScript to execute System commands.
```

Reply:
instead of `System` should it be `Root`?

Reply:
no, by system commands I mean any command such as whoami cp touch etc..